docs/UserGuide.adoc

```diff
@@ -474,35 +493,53 @@ Examples:
 * `*cache-delete* 2` +
 Deletes the offline copy of the second bookmark.
 
+[[annotate]]
 === Making annotations on an offline copy: *`annotate`*
 
-This adds a new annotation on the offline copy of the given bookmarked
-website. When you give this command, the UI will switch to the offline tab showing the results of your command.
-With this command, you can highlight a paragraph on the offline document and optionally attach a supplementary
-note to said paragraph. (You can add notes to justify the highlight or as
-content-relevant notes to refer to in future).
-You can also add a general note not pertaining to any specific paragraph to the bottom of the document,
-hereby known as the _General notes section_.
+If you want to add a new annotation on the offline copy of a bookmark,
+you can do so using the `annotate` command.
+With this command, you can highlight a paragraph on the offline document and
+optionally attach a supplementary note to said paragraph. You can add notes
+to justify the highlight or as content-relevant notes to refer to in future.
 
-NOTE: A general note is known to attach to a <<phantom-paragraph,_phantom paragraph_>>. As you can see in the offline
-copy, phantom paragraphs do not have any content displayed; they do not belong to the original web page.
-The phantom paragraphs altogether make up the _General notes section_.
+If you are looking to add a general note not pertaining to any specific paragraph, you
+can also use this command to add it to the bottom of the page, hereby known as the _General notes section_.
+
+Upon annotating, the your view will be switched to the offline tab
+showing the results of your command.
+
+NOTE: A general note is not attached to any paragraph of the original web page. Instead, when adding general notes, they are attached
+to newly created paragraphs that do not have any content displayed.
+General notes are found at the bottom of the page, referred to as the _General notes section_.
 
 You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
 a phantom paragraph, Mark remembers the hidden highlight colour, but no highlight is reflected on the application.
```

Comment:
I saw you removed the phantom paragraph from the glossary should update here as well?

Reply:
Thanks for checking; I'll change the 'phantom paragraph' in the next paragraph into 'paragraph of general notes'